apps/mobile/src/components/ModifyOffer/atoms/offerFormStateAtoms.ts

```diff
@@ -264,16 +266,23 @@ export const offerFormMolecule = molecule((getMolecule, getScope) => {
       set(deletingOfferAtom, true)
       return pipe(
         set(deleteOffersAtom, {
-          adminIds: [offer.ownershipInfo?.adminId ?? ('' as OfferAdminId)],
+          adminIds: [offer.ownershipInfo?.adminId].filter(notEmpty),
         }),
         TE.match(
           (e) => {
             Alert.alert(
-              toCommonErrorMessage(e, t) ?? t('offerForm.errorCreatingOffer')
+              toCommonErrorMessage(e, t) ?? t('editOffer.errorDeletingOffer')
             )
             return false
           },
           (result) => {
+            set(offerToConnectionsAtom, (previousValue) => ({
```

Comment:
How about moving this to deleteOffersAtom set action? It might be better idea since with every offer deletion we also want to delete the offer from offer to connections. What do you think?